lizmap/modules/view/locales/en_US/dictionnary.UTF-8.properties

```diff
@@ -226,6 +226,7 @@ digitizing.toolbar.import=Import in GeoJSON, GPX, KML, zipped Shapefile or FlatG
 digitizing.toolbar.edit=Edit (hold "Shift" to select multiple features, hold "Alt" and click to delete a vertex)
 digitizing.toolbar.split=Split
 digitizing.toolbar.rotate=Rotate
+digitizing.toolbar.scaling=Scaling
```

Comment:
```suggestion
digitizing.toolbar.scaling=Scale the geometry
```

Reply:
Agreed, similar patterns as other verbs `Split`, `Rotate` ...

Please add the keyboard shorts which are available.

> (hold "Shift" to lock proportions)

Tested locally

Reply:
Done

Reply:
Question, why not keeping the keyboard hint I have proposed ?